compiler/src/dotty/tools/repl/ReplCompiler.scala

```diff
@@ -88,6 +88,16 @@ class ReplCompiler extends Compiler {
     var valIdx = state.valIndex
     val defs = new mutable.ListBuffer[Tree]
 
+    /** If the user inputs a definition that's name is of the form REPL_RES_PREFIX and a number,
```

Comment:
```suggestion
    /** If the user inputs a definition whose name is of the form REPL_RES_PREFIX and a number,
```
No?

Reply:
It sounded so wrong, but then I though "whose" was for people not things.